lib/bas/serialize/imap/support_emails.rb

```diff
@@ -3,10 +3,10 @@
 require_relative "../../domain/email"
 require_relative "../base"
 
-module Mapper
+module Serialize
   module Imap
     ##
-    # This class implementats the methods of the Mapper::Base module, specifically designed for
+    # This class implementats the methods of the Serialize::Base module, specifically designed for
```

Comment:
Same as above, an other modules with the same issue 😅 
```suggestion
    # This class implements the methods of the Serialize::Base module, specifically designed for
```